proto/sequencerblockapis/astria/sequencerblock/v1alpha1/block.proto

```diff
@@ -0,0 +1,136 @@
+syntax = "proto3";
+
+package astria.sequencerblock.v1alpha1;
+
+import "astria/primitive/v1/types.proto";
+import "astria/sequencerblock/v1alpha1/merkle.proto";
+import "google/protobuf/timestamp.proto";
+
+// `RollupTransactions` are a sequence of opaque bytes together with a 32 byte
+// identifier of that rollup.
+//
+// The binary encoding is understood as an implementation detail of the
+// services sending and receiving the transactions.
+message RollupTransactions {
+  // The 32 bytes identifying a rollup. Usually the sha256 hash of a plain rollup name.
+  bytes id = 1;
```

Comment:
Can we name this `rollup_id` to make the other names? (like `rollup_ids_proof` in the block, and `rollup_id` in `CelestiaRollupBlob`)